redash/cli/database.py

```diff
@@ -7,6 +7,8 @@
 def create_tables():
     """Create the database tables."""
     from redash.models import db
+    # make sure the additional search mappers are configured
+    db.configure_mappers()
```

Comment:
What are the search mappers?

Reply:
The comment is misleading, this basically makes sure that all mappers that have been registered with the SQLAlchemy ORM have been properly configured. Since SQLAlchemy-Searchable installs database triggers that have to act on when ORM model instances are created (e.g. to track updates), this call is needed.

Here's what the SA-Searchable docs say:

> It is very important here that you either access your searchable class or call configure_mappers before the creation of tables. SA-Searchable adds DDL listeners on the configuration phase of models.

Since this code happens in the phase in which tables are created, we can't really access the searchable class, but have to call `configure_mappers`. I'll update the comment to reflect this.